### PR TITLE
refactor: update translates [WEB-1448]

### PR DESCRIPTION
**Pull request by turtlemoji**
## Description
Setup for new locize project

## Motivation and Context
Enabled the locize project again so we can add missing langs

## How Has This Been Tested?
PR Test for en and ES under pr preview and local

### CODE REVIEW

File: public/locales/en/ironbank.json [hunk 1]
```diff
@@ -7,11 +13,5 @@
...
     "text": "Check back soon."
-  },
-  "ironbank-announcement-card": {
-    "header": "To our Iron Bank users",
-    "desc-1": "The Iron Bank will now operate as an independent project with its own communication channels and frontend. You will find your funds safe and available through their UI [app.ib.xyz](https://app.ib.xyz).",
-    "desc-2": "This decision provides the clarity that our communities need while enabling both parties to make decisions and move forward more quickly to build the future of DeFi.",
-    "desc-3": "Full announcement [here](https://medium.com/iearn/clarifying-2020-mergers-an-independent-iron-bank-a6f8f3f4c25e)."
   }
-}
+}
```
Comment (karelianpie) [inline public/locales/en/ironbank.json:17]: nit: Quite a few newlines missing, probably worth to add them in

**Review by karelianpie — APPROVED** [review]
LGTM, just a lot of newlines missing -- wondering if prettier is not configured correctly for .json files

File: public/locales/en/ironbank.json (hunk 1 above)
Comment (turtlemoji) [reply public/locales/en/ironbank.json:17]: it is an auto generated file from locsize now

**Comment by turtlemoji** [comment]
> LGTM, just a lot of newlines missing -- wondering if prettier is not configured correctly for .json files

Yeah, locsize formats the json differently, his is the result of "yarn downloadProdLocales"

**Review by xgambitox — APPROVED** [review]

**Review by FoxTheSin — APPROVED** [review]

**Pull request merged**